src/vhdldocgen.cpp

```diff
@@ -200,7 +205,7 @@ void VhdlDocGen::writeOverview()
 
   if (!f.open(IO_WriteOnly))
   {
-    err("Warning: Cannot open file %s for writing\n",fileName.data());
+    fprintf(stderr,"Warning: Cannot open file %s for writing\n",fileName.data());
```

Comment:
Why is `err` replaced by `fprintf(stderr`, the `err` is the default way doxygen handles error

Reply:
OOps that was my mistake. 
err is correct